src/component/veilederverktoy/start-eskalering/start-eskalering.tsx

```diff
@@ -65,6 +68,8 @@ function StartEskalering() {
                         startEskaleringPromise,
                     ]);
 
+                    logger.event('visittkort.metrikker.forhonshorendtering.sendt', { type: values.type });
```

Comment:
Vi har brukt 'veilarbvisittkortfs' som prefix for de fleste metrikkene, kan være greit å gjøre det her også